code/app/src/main/java/com/example/sigma_blue/Item.java

```diff
@@ -1,10 +1,11 @@
 package com.example.sigma_blue;
 
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Date;
 
-public class Item implements Comparable<Item>{
+public class Item implements Comparable<Item>, Serializable {
```

Comment:
We will want to swap this to parcelable when we have the chance. Serializable is slower due to more IO calls.

Reply:
I did this so Item was capable of being inserted inside of a Bundle. Do we want to do the switch then for all objects we wish to bundle?

Reply:
> I did this so Item was capable of being inserted inside of a Bundle. Do we want to do the switch then for all objects we wish to bundle?

I would say once we start integrating all the activities together, I might change some of these to parcelable. I think it would be good if we start implementing parcelable instead of serializable now, but it's not customer urgent.